# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/AnimationSurfaceProvider.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c image/AnimationSurfaceProvider.cpp
--- a/image/AnimationSurfaceProvider.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/image/AnimationSurfaceProvider.cpp	Fri May 24 11:32:54 2019 +0000
@@ -5,7 +5,7 @@
 
 #include "AnimationSurfaceProvider.h"
 
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/gfx/gfxVars.h"
 #include "nsProxyRelease.h"
 
@@ -464,7 +464,7 @@
   MutexAutoLock lock(mDecodingMutex);
   MOZ_ASSERT(mDecoder);
 
-  return mDecoder->ShouldSyncDecode(gfxPrefs::ImageMemDecodeBytesAtATime());
+  return mDecoder->ShouldSyncDecode(StaticPrefs::ImageMemDecodeBytesAtATime());
 }
 
 RawAccessFrameRef AnimationSurfaceProvider::RecycleFrame(